config/index.js

```diff
@@ -20,7 +20,7 @@ const locals = {
   title: 'Etcher',
   prefix: IS_PROD ? '' : '',
   slogan: 'Burn. Better.',
```

Comment:
Not sure what to do about this one – I vaguely recall this was discussed at some point, but not sure if we have a new wording for this now? /cc @alisondavis17

Reply:
I have nothing off the top of my head but happy to discuss it if needed.